packages/api/test/integration/data.contracts.spec.js

```diff
@@ -290,21 +290,8 @@ describe('DataContracts routes', () => {
       assert.deepEqual(body, expectedDataContract)
     })
 
-    //
```

Comment:
Don't delete this test, I think I forgot to reimplement that during one of the refactors

Reply:
✅